src/workos.spec.ts

```diff
@@ -222,6 +224,30 @@ describe('WorkOS', () => {
       });
     });
 
+    describe('when the api responses with a 429', () => {
+      it('throws a RateLimitExceededException', async () => {
+        fetchOnce(
+          {
+            message: 'Too many requests',
```

Comment:
Is this verbatim what we return from the API? I'd like there to be more information in here, like a link to the [rate limiting docs](https://workos.com/docs/reference/rate-limits) or a callout to reach out on support@ if they want to discuss higher rate limits.

Reply:
It is, and agreed that we should have more information in the `message`. Separately, I can look into updating that, adding the additional details you mentioned, so that all SDK's (or no SDK at all) will have the extra context.